skuba-update/skuba_update/skuba_update.py

```diff
@@ -315,7 +447,7 @@ def run_zypper_command(command, needsOutput=False):
 
     process = run_command(zypperCommand, needsOutput)
     if is_zypper_error(process.returncode):
-        raise Exception('"{0}" failed'.format(' '.join(zypperCommand)))
+        raise ZypperRunError('"{0}" failed'.format(' '.join(zypperCommand)))
```

Comment:
Side note for reviewers: This is necessary to avoid catching _all_ exceptions above, but also it's a better practice to send custom exceptions.